scripts/download_backtest_data.py

```diff
@@ -27,15 +30,36 @@
 
 print(f'About to download pairs: {PAIRS} to {dl_path}')
 
-# Init Bittrex exchange
+# Init exchange
 exchange._API = exchange.init_ccxt({'key': '',
                                     'secret': '',
-                                    'name': 'bittrex'})
+                                    'name': args.exchange})
 
 for pair in PAIRS:
     for tick_interval in TICKER_INTERVALS:
         print(f'downloading pair {pair}, interval {tick_interval}')
-        data = exchange.get_ticker_history(pair, tick_interval)
+
+        since_time = None
+        if args.days:
+            since_time = int((time.time() - args.days * DAY_SECCONDS) * 1000)
+
+        # download data until it reaches today now time
+        data = []
+        while not since_time or since_time < (time.time() - 10 * MIN_SECCONDS) * 1000:
+            data_part = exchange.get_ticker_history(pair, tick_interval, since=since_time)
+
+            if not data_part:
+                print('\tNo data since %s' % datetime.datetime.utcfromtimestamp(since_time / 1000).strftime('%Y-%m-%dT%H:%M:%S'))
```

Comment:
I know flake8 is not checked in the scripts-directory ... but i still think this line is too long and we should stick to the 100 char limit